test-cases/performance/perf-regression-gradual-throughput-grow-2.3tb.yaml

```diff
@@ -0,0 +1,65 @@
+test_duration: 500
+prepare_write_cmd: [
+#  "cassandra-stress write no-warmup cl=ALL n=5000000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode connectionsPerHost=8 cql3 native -rate threads=150 -col 'size=FIXED(1024) n=FIXED(1)'  -pop seq=1..5000000",
```

Comment:
what those commented out command ?

Reply:
I keep this for testing of code changes. When I do not need to run full test.
But I can use `extra_environment_variables` as well

Reply:
I would suggest doing that in it's own commit for testing, otherwise it's not clear for reviews

and we can push it in, by mistake